### ISSUE TITLE
Adding 2nd endpoint of Aqara device to a group is failing on a timeout

### ISSUE THREAD

**Issue by pkedvessy** [Koenkk/zigbee2mqtt#5845]
## What happened
I tried to add 2nd endpoint of Aqara wall switch to a group but it is failing on a timeout. I have tried to add it both from the UI and by an MQTT message but the result is the same. I have tried to use the following device names but all of them failed:
* <friendly_name>/right (as per the documentation)
* <friendly_name>/2
* < id >/right
* < id >/2

Left channel (1st endpoint) is working as expected.

I experienced exactly the same with QBKG24LM and QBKG12LM Aqara wall switches.

Console log: https://pastebin.com/hVNfmPVi

## What did you expect to happen
Having the 2nd endpoint (right channel) in the group as well as the 1th (left channel).

## How to reproduce it (minimal and precise)
Add a member to a group with non default endpoint by mqtt message like
`{"group":"my_group","device":"<friendly_name>/right"}`

Add 2nd endpoint to a group by selecting it on the UI.

## Debug info
Zigbee2MQTT version: 1.17.0 (1.16 has the same issue)
Adapter hardware: CC1352P-2
Adapter firmware version: 20200417

**Comment by Koenkk** [comment]
Duplicate of https://github.com/Koenkk/zigbee2mqtt/issues/4622 

TLDR; second endpoint doesn't support it
